index.js

```diff
@@ -0,0 +1,104 @@
+import { addMenuLinks } from './modules/displaymenu.js';
+import { displayTime } from './modules/date.js';
+import Book from './modules/class-book.js';
+
+const removeBookFromDOM = (listBooks, id) => {
+  if (id !== -1) {
+    listBooks.removeBook(id);
+  }
+};
+
+const addRemoveButtons = (listBooks) => {
+  const buttons = document.getElementsByClassName('btn remove-btn');
+  for (let i = 0; i < buttons.length; i += 1) {
+    buttons[i].addEventListener('click', () => {
+      removeBookFromDOM(listBooks, i);
+    });
+  }
+};
```

Comment:
- [ ] Good job implementing the remove book functionality, but the problem is that the book is not getting removed upon clicking until the user refreshes the page. kindly consider fixing it for a pleasant user experience.